Ej15.rb

```diff
@@ -0,0 +1,31 @@
+puts 'Welcome'
+puts 'dias convertidos a años,meses,semanas,dias'
+puts 'Ingrese el numero de dias'
+n=gets.to_i
+anos=0
+meses=0
+semanas=0
+
+dias=n
+
+while dias>365
```

Comment:
Again, use math expressions here. Refer to the comment I made on the ATM problem.